README.md

```diff
@@ -1,4 +1,5 @@
 # VPN Client
+[![Build Status](https://travis-ci.org/agentcobra/vpnclient_ynh.svg?branch=master)](https://travis-ci.org/agentcobra/vpnclient_ynh)
```

Comment:
Please use `https://travis-ci.org/labriqueinternet/vpnclient_ynh` and not your username :)